libs/components/forms/src/lib/modules/file-attachment/file-attachment.component.ts

```diff
@@ -107,6 +107,14 @@ export class SkyFileAttachmentComponent
   @Input()
   public helpPopoverTitle: string | undefined;
 
+  /**
+   * Unique key specifying the page the help panel opens to. When specified a [help inline](https://developer.blackbaud.com/skyux/components/help-inline) button is
```

Comment:
This description is only accurate for internal Blackbaud applications. External consumers will have to implement their own method for showing global help which may or may not display a help panel like Blackbaud's. I suggest something more like this:

"A unique key that identifies the global help content to display. When specified a [help inline](https://developer.blackbaud.com/skyux/components/help-inline) button is added to the single file attachment label. Clicking the help inline button will invoke global help as configured by the application."

We can get more specific about the "as configured by the application" part when we document the new `SkyHelpService`.

Reply:
Looks good and is similar to a rewrite I was working on after Sandhya pinged me. I have a few follow-up questions/suggestions...

- In the first sentence, is "unique" accurate? I mean, it seems possible that we could have a generic topic about file attachments and then reuse the help key across multiple instances of the component. Should we change "unique" to "help" to account for that?
- In the second sentence, we should have a comma after "specified"
- Also in the second sentence, should "is added to" be "appears beside" or "is placed beside" instead? Might not matter, but it seemed like "added" could suggest that the button is part of the label.
- In the third sentence, I think we can delete "help inline" before button because it's a second reference.
- Also in the third sentence, I'd change "will invoke" to "invokes"

Reply:
I was thinking "unique" as in each help topic's unique identifier is the help key, but is probably just confusing in this context. I'm fine with removing it. Your other suggestions are fine as well.

Reply:
🫡